plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -524,6 +526,35 @@ def updateGUIFromParameterNode(self, caller=None, event=None):
         idx = 0 if idx < 0 < self.ui.labelComboBox.count else idx
         self.ui.labelComboBox.setCurrentIndex(idx)
 
+        if self._segmentNode:
```

Comment:
this logic can be combined with above logic where we populate items for self.ui.labelComboBox

Reply:
addressed in latest commit.